include/seqan3/search/dream_index/concept.hpp

```diff
@@ -0,0 +1,41 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Enrico Seiler <enrico.seiler AT fu-berlin.de>
+ * \brief Provides concepts for the seqan3::dream_index module.
+ */
+
+#pragma once
+
+#include <seqan3/core/metafunction/template_inspection.hpp>
+#include <seqan3/range/container/concept.hpp>
+#include <seqan3/search/dream_index/binning_directory.hpp>
```

Comment:
why do you include the binning_directory here?
also why the container/concept?

Reply:
sdsl/bit_vector is probably enough
The concept was needed before